index.html

```diff
@@ -188,18 +194,92 @@ <h3 class="projects-header">Multi-Post Stories</h3>
             </p>
           </div>
           <div class="projects__languages">
-            <ul class="projects__languages-list">
-              <li class="projects__languages-first">css</li>
-              <li class="projects__languages-second">html</li>
-              <li class="projects__languages-third">bootstrap</li>
-              <li class="projects__languages-fourth">ruby</li>
+            <ul class="projects__languages--list">
+              <li class="projects__languages--first">css</li>
+              <li class="projects__languages--second">html</li>
+              <li class="projects__languages--third">bootstrap</li>
+              <li class="projects__languages--fourth">ruby</li>
             </ul>
           </div>
           <div>
-            <button class="btn">See Projects</button>
+            <button type="button" class="btn">See Projects</button>
           </div>
         </div>
       </section>
+      <section class="contact">
+        <div class="contact__text">
+          <h3 class="contact__text--header">Contact Us</h3>
+          <p class="description__text">
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it
+          </p>
+        </div>
+        <div>
+          <form
+            class="form"
+            action="https://formspree.io/f/xeqnpand"
+            method="POST"
+          >
+            <input
+              type="text"
+              name="name"
+              id="name"
+              placeholder="Text Input"
```

Comment:
[OPTIONAL] your input fields should have the correct associated placeholder text. In your case, the placeholder text should be `name` instead of `Text input`.

Reply:
Thanks so much